package/model-tflite-mlperf-ssd-mobilenet/.cm/meta.json

```diff
@@ -0,0 +1,45 @@
+{
+  "check_exit_status": "yes", 
+  "customize": {
+    "install_env": {
+      "MODEL_TFLITE_GRAPH": "detect.tflite",
+      "CK_MODEL_DATA_LAYOUT": "NHWC", 
```

Comment:
@ens-lg4 As you introduced this variable for all models, is there any reason for the `CK_MODEL` prefix here? (It just looks odd in the company of variables with the `MODEL` prefix.)